docs/userguide/networking/dockernetworks.md

```diff
@@ -485,17 +489,27 @@ networks can include features not present in Docker's default networks. For more
 information on writing plugins, see [Extending Docker](../../extend/index.md) and
 [Writing a network driver plugin](../../extend/plugins_network.md).
 
-## Legacy links
+### Docker Embedded DNS server
+
+Docker daemon runs an embedded DNS server to provide automatic service discovery
+for containers connected to user defined networks. Name resolution requests from
+the containers are handled first by the embedded DNS server. If the embedded DNS
+server is unable to resolve the request it will be forwarded to any external DNS
+servers configured for the container. To facilitate this when the container is
+created, only the embedded DNS server reachable at 127.0.0.11 will be listed in the
+container's resolv.conf file. More information on embedded DNS server on user-defined
+networks can be found [here](configure-dns.md)
```

Comment:
Trying to avoid using "here" as caption for links, so e.g.;
```
More information on the embedded DNS server on user-defined
networks can be found in the [embedded DNS server in user-defined networks](configure-dns.md] section.
```